.vscode/launch.json

```diff
@@ -0,0 +1,67 @@
+{
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Debug Server",
+			"type": "go",
+			"request": "launch",
+			"mode": "exec",
+			"program": "${workspaceFolder}/temporal-server",
+			"args": [
+				"start",
+			]
+		},
+		{
+			"name": "Debug CLI Namespace Describe",
+			"type": "go",
+			"request": "launch",
+			"mode": "exec",
+			"program": "${workspaceFolder}/tctl",
+			"args": [
+				"--ns",
```

Comment:
Is it all for demo purpose? I am not sure if we need to specify `default` namespace.

Reply:
yes just a sample

Reply:
Can we just have a single config for cli instead of 3 separate ones?

Reply:
i removed the global namespace argument from all commands except the first one. Also renamed in the first one from --ns to --namespace since it's meant to be an example for new to temporal coders